cram/cram_io.c

```diff
@@ -4575,3 +4575,27 @@ int cram_set_voption(cram_fd *fd, enum hts_fmt_option opt, va_list args) {
 
     return 0;
 }
+
+#define TEMPLATE_LEN 38
+int cram_check_EOF(cram_fd *fd)
+{
+    uint8_t buf[TEMPLATE_LEN];
```

Comment:
This also came from BGZF implementation. I guess it depends on how likely it is that someone compiles on a DSP where CHAR_BIT > 8 (<8 is ruled out in C90). How much does this bug you will change if lots?